modules/git/repo_language_stats_gogit.go

```diff
@@ -45,6 +45,7 @@ func (repo *Repository) GetLanguageStats(commitID string) (map[string]int64, err
 	defer deferable()
 
 	sizes := make(map[string]int64)
+	explicitIncludedLanguage := make(map[string]struct{})
```

Comment:
Should this PR be blocked until #21408 is merged?

Reply:
I will update the (other) PR whichever gets merged first.

Reply:
done

Reply:
#21408 merged